lib/fenrir_view/components/fenrir_view_properties/fenrir_view_properties.yml

```diff
@@ -0,0 +1,30 @@
+meta:
+  status: "work in progress" # 'deprecated', 'work in progress', 'in review', 'in use'
+  description: "Table for displaying component properties."
+  devices:
+  - "800x200"
+
+stubs:
+- name: "Component Properties"
+  devices:
+  - "1100x500"
+  examples:
+  - name: "Basic Property List"
+    properties:
+      property_list:
+        Ruby:
+          required: true
+          one_of_type:
+            - 'Bool'
+          default: true
+        JavaScript:
```

Comment:
Could we add an additional test case for properties that has a symbol value?

I believe the current implementation of the properties table renders symbols as strings which is confusing as `symbol` and `:symbol` won't be the same thing...